### DISCUSSION TITLE
Markdown citations interpreted as list headings

### DISCUSSION THREAD

**Discussion by malcolmsailor** [jgm/pandoc#10284] [Q&A]
In Pandoc's Markdown, I'm having an issue with citations after newlines in list items being interpreted as list headings. Here's what seems to be a minimal example:

```
- Miscellaneous text in list followed by line break and citation
    @author2024paper.
```

If I save this in `file.md` and run `pandoc -t latex file.md`, I get the following output:

```
\begin{itemize}
\tightlist
\item
  Miscellaneous text in list followed by line break and citation

  \begin{enumerate}
  \def\labelenumi{\arabic{enumi}.}
  \tightlist
  \item
  \end{enumerate}
\end{itemize}
```

The citation is rendered as a list item, which is of course not what I want.

If I put the citation in curly braces like `@{author2024paper}`, it fixes the issue. However, my workflow involves first recompiling my document as markdown in order to apply a few extensions (i.e., doing `pandoc -f markdown -t markdown+EXTENSIONS`) and at this stage pandoc removes the curly braces.

Is there an approved way of avoiding this scenario?

**Comment by jgm** [comment]
It is, in fact, valid syntax for an example list item.

I don't have any good advice.  Why don't you open issue requesting that the markdown writer use `{}` for citations when they begin a line?

**Reply by jgm** [reply]
On second thought, I don't see a clean way of implementing this.

**Comment by malcolmsailor** [comment]
Thanks for your reply.

I figured out a workaround for my use case which is to use the `--wrap=preserve` option when doing the `-f markdown -t markdown` step. The hard line breaks pandoc was adding was what was sometimes placing my citations at the start of lines in this way.
